source/sass/component/_image.scss

```diff
@@ -1,42 +1,45 @@
 $c-image-border-radius: var(--c-image-border-radius, $border-radius-xs) !default;
 
-
 // ***************************************************************
 // *** Mapping for different sizes
 // ***************************************************************
 $radius: (
     (
-        "name": "",
-        "size": $border-radius-md
+        'name': '',
+        'size': $border-radius-md,
     ),
     (
-        "name": &--rounded-xs,
-        "size": $border-radius-xs
+        'name': &--rounded-xs,
+        'size': $border-radius-xs,
     ),
     (
-        "name": &--rounded-sm,
-        "size": $border-radius-sm
+        'name': &--rounded-sm,
+        'size': $border-radius-sm,
     ),
     (
-        "name": &--rounded-md,
-        "size": $border-radius-md
+        'name': &--rounded-md,
+        'size': $border-radius-md,
     ),
     (
-        "name": &--rounded-lg,
-        "size": $border-radius-lg
+        'name': &--rounded-lg,
+        'size': $border-radius-lg,
     )
 );
 
 // ***************************************************************
 // *** Placeholder
 // ***************************************************************
 
+.modularity-mod-image .c-link {
```

Comment:
Not a valid namespace for styleguide. This fix should be placed in modularity. I would suggest to create a utility in the styleguide, and apply this trough the classList argumment to the component.